### PR TITLE
Added English Quotes

### PR DESCRIPTION
**Pull request by TheNatBoi**
### Description

Adding various English quotes

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4944243637

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4944278798

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -39155,6 +39155,204 @@
...
+      "text": "Don't throw away the old bucket until you know whether the new one holds water.",
+      "source": "Swedish Proverb",
+      "length": 79,
+      "id": 6916 
+    },
+    {
+      "text": "It is a sad fact that 50 percent of marriages in this country end in divorce. But hey, the other half end in death. You could be one of the lucky ones!",
+      "source": "Richard Jeni",
+      "length": 151,
+      "id": 6917
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39193]: Remove

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -39155,6 +39155,204 @@
...
+    },
+    {
+      "text": "For four-fifths of our history, our planet was populated by pond scum.",
+      "source": "J. W. Schopf",
+      "length": 70,
+      "id": 6918
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39199]: Remove

File: frontend/static/quotes/english.json [hunk 3]
```diff
@@ -39155,6 +39155,204 @@
...
+      "text": "Don't think there are no crocodiles because the water is calm.",
+      "source": "Malayan Proverb",
+      "length": 62,
+      "id": 6941
+    },
+    {
+      "text": "Don't be sweet, lest you be eaten up; don't be bitter, lest you be spewed out.",
+      "source": "Jewish Proverb",
+      "length": 78,
+      "id": 6942
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39343]: Remove

File: frontend/static/quotes/english.json [hunk 4]
```diff
@@ -39155,6 +39155,204 @@
...
+    {
+      "text": "Keep five yards from a carriage, ten yards from a horse, and a hundred yards from an elephant; but the distance one should keep from a wicked man cannot be measured.",
+      "source": "Indian Proverb",
+      "length": 165,
+      "id": 6943
+    },
+    {
+      "text": "Rejoice not at thine enemy's fall - but don't rush to pick him up either.",
+      "source": "Jewish Proverb",
+      "length": 73,
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39354]: Remove

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/quotes/english.json (hunk 1 above)
Comment (TheNatBoi) [reply frontend/static/quotes/english.json]: Fixed

File: frontend/static/quotes/english.json (hunk 3 above)
Comment (TheNatBoi) [reply frontend/static/quotes/english.json]: Fixed

**Comment by Miodec** [comment]
Please resove merge conflicts

**Comment by Miodec** [comment]
I think something is messed up because the only file changes i see now is just the french quotes file.

**Comment by TheNatBoi** [comment]
im just gonna close this and open another one bc i have no clue what its doing

**Pull request closed without merging**